### ISSUE TITLE
[BUG] NVRTC error: NVRTC_ERROR_INVALID_OPTION when doing sort_values

### ISSUE THREAD

**Issue by drabastomek** [rapidsai/cudf#2712]
**Description**
On the latest Docker Image with cudf v 0.9 every time I try to do sort_values it throws an NVRTC_ERROR_INVALID_OPTION error.

**Steps/Code to reproduce bug**
```
import pandas as pd
import cudf

gpu_df = cudf.from_pandas(
    pd.DataFrame({
          'id': [1,2,4,6,7,3]
        , 'val': [66,2345,789,11,53,987]
    })
)
gpu_df.sort_values('id')
```
**Expected behavior**
A sorted cudf should be returned, sorted by the `'id'` column

**Environment overview (please complete the following information)**
 - Environment location: Docker image a0036af0fdcb
 - Method of cuDF install: Docker
   - `docker pull rapidsai/rapidsai:latest`
   - `docker run -d --runtime=nvidia -it -v /home/tom/Documents/programming/:/rapids/notebooks/host_folder -p 8888:8888 -p 8787:8787 -p 8786:8786 a0036`

**Additional context**
Looks like the bug is in the `cudf.bindings.binops.apply_op()`.

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-36-76a6edc2f3fd> in <module>
      8     })
      9 )
---> 10 gpu_df.sort_values('id')

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/dataframe.py in sort_values(self, by, ascending, na_position)
   1938         # argsort the `by` column
   1939         return self.take(
-> 1940             self[by].argsort(ascending=ascending, na_position=na_position)
   1941         )
   1942 

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/series.py in argsort(self, ascending, na_position)
   1392         result: Series
   1393         """
-> 1394         return self._sort(ascending=ascending, na_position=na_position)[1]
   1395 
   1396     def sort_index(self, ascending=True):

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/series.py in _sort(self, ascending, na_position)
   1466         """
   1467         col_keys, col_inds = self._column.sort_by_values(
-> 1468             ascending=ascending, na_position=na_position
   1469         )
   1470         sr_keys = self._copy_construct(data=col_keys)

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/numerical.py in sort_by_values(self, ascending, na_position)
    152     def sort_by_values(self, ascending=True, na_position="last"):
    153         sort_inds = get_sorted_inds(self, ascending, na_position)
--> 154         col_keys = cpp_copying.apply_gather(self, sort_inds.data.mem)
    155         col_inds = self.replace(
    156             data=sort_inds.data,

cudf/bindings/copying.pyx in cudf.bindings.copying.apply_gather()

cudf/bindings/copying.pyx in cudf.bindings.copying._normalize_maps()

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/numerical.py in binary_operator(self, binop, rhs, reflect)
     73                 op=binop,
     74                 out_dtype=out_dtype,
---> 75                 reflect=reflect,
     76             )
     77         else:

/opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/dataframe/numerical.py in numeric_column_binop(lhs, rhs, op, out_dtype, reflect)
    399     out = columnops.column_empty(row_count, dtype=out_dtype, masked=masked)
    400     # Call and fix null_count
--> 401     null_count = cpp_binops.apply_op(lhs, rhs, out, op)
    402 
    403     if is_op_comparison:

cudf/bindings/binops.pyx in cudf.bindings.binops.apply_op()

cudf/bindings/binops.pyx in cudf.bindings.binops.apply_op_v_s()

RuntimeError: NVRTC error: NVRTC_ERROR_INVALID_OPTION
```

**Comment by harrism** [comment]
@devavret can you take a look?

**Comment by devavret** [comment]
@harrism This should be fixed with the new version of Jitify. I’ll check with ops to use the latest branch.
@drabastomek Meanwhile, can you post your CUDA version and device model? Just to check if it indeed is the bug that I think it is.

**Comment by drabastomek** [comment]
@devavret This was tested on CUDA 10.0 and Titan RTX. I'm about to test it on CUDA 10.1 -- will let  you know.

**Comment by drabastomek** [comment]
Just tested on a host with CUDA 10.1 (and the latest Docker image from rapidsai/rapidsai) and got the same error.

**Comment by drabastomek** [comment]
Any groupby operations also render the same error as these call the sort_values(...) as well.

**Comment by devavret** [comment]
@drabastomek The issue is with using CUDA 9.2 with a Turing architecture device. Even though the CUDA version in your machine is 10, the one in the docker image is 9.2. The bug originates in a submodule we use and it has been fixed there. I'm testing the latest update for that submodule. That should fix it.

**Comment by drabastomek** [comment]
Got ya. I didn't update CUDA in the docker image (nor have I checked that version, tbh...). Awesome! I'm assuming you'll update the docker once this is merged for 0.9? Or will this become RAPIDS 0.10?

**Comment by devavret** [comment]
@drabastomek 0.9 is released so this fix will go into 0.10. But you can unblock yourself right now with a different docker image. Check out any of the images with CUDA 10 here: https://hub.docker.com/r/rapidsai/rapidsai. This one should work: `cuda10.0-runtime-ubuntu16.04`

**Comment by drabastomek** [comment]
Awesome! Thanks!

**Comment by harrism** [comment]
Fixed by rapidsai/jitify#1
